src/scss/components/switch/mixins/_hxSwitchFacade.scss

```diff
@@ -0,0 +1,61 @@
+//
+// ===== Mixins for <hx-checkbox> facade states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __checkboxFacade($pseudo-state: null) {
+  @if $pseudo-state == focus {
+    border-color: $cyan-700;
+    box-shadow: $focus-glow;
+  }
+  @else if $pseudo-state == hover {
+    background-color: $cyan-50;
+    border-color: $cyan-500;
+    color: $cyan-500;
+  }
+}
+
+// ========== INVALID ==========
+@mixin __checkboxFacade--invalid($pseudo-state: null) {
+  @if $pseudo-state == focus {
+    border-color: $red-900;
+    box-shadow: $focus-glow-invalid;
+  }
+  @else if $pseudo-state == hover {
+    background-color: $red-100;
+    border-color: $red-900;
+    color: $red-900;
+  }
+  @else {
+    border: 2px solid $red-900;
+    color: $red-900;
+  }
+}
+
+// ========== DISABLED ==========
+@mixin __checkboxFacade--disabled($pseudo-state: null) {
```

Comment:
Please change to:

```scss
@mixin __switchFacade--disabled($pseudo-state: null) {
```

We will have to confirm if these these are `<hx-checkbox>` or `<hx-switch>` styles, and also confirm that they are used.

Reply:
As facade mixins are not used , i removed this file